### DISCUSSION TITLE
official docker image release?

### DISCUSSION THREAD

**Discussion by StroepWafel** [pelican-dev/panel#842] [Ideas]
will there ever be an official release of the docker image? if so, is there an ETA?
cheers

**Comment by Boy132** [comment]
There already are docker images:
`ghcr.io/pelican-dev/panel`
`ghcr.io/pelican-dev/wings`

**Reply by StroepWafel** [reply]
what about docker compose files?
are there any? 
i could not find them if there are.

**Reply by Boy132** [reply]
https://github.com/pelican-dev/panel/blob/main/compose.yml
https://github.com/pelican-dev/wings/blob/main/docker-compose.example.yml

**Reply by StroepWafel** [reply]
ah, cheers

**Comment by appreciated** [comment]
@Boy132 Is there a specific reason why the images / docker compose files are not being mentioned in the official installation guide under:
- https://pelican.dev/docs/panel/getting-started
- https://pelican.dev/docs/wings/install

**Reply by Boy132** [reply]
There will be a separate documentation for the dockerized install method. The dockerized install will also be the "default" way of installing at some point.

Also, the docker images still need some more tweaking.
